script.js

```diff
@@ -1,7 +1,102 @@
-console.log("hello script js");
+// UI elements
+const inputTemp = document.querySelector('#temp-input');
+const inputUnit = document.querySelector('#input-unit');
 
-var inputHappened = function(currentInput){
-  console.log( currentInput );
-  var output = "WOW TEMPERATURE";
-  return output;
+// convert button
+document.querySelector('#btn').addEventListener('click', function(){
+  let username = document.querySelector('#name').value;
+  if (username === "") {
+    username = "John Doe";
+  } else {
+    username = document.querySelector('#name').value;
+  }
+
+  // check for input temperature and unit
+  if (inputTemp.value === "" || inputUnit.value === "") {
+    const output = "Please enter a temperature";
+    displayOne(output);
+  } else 
+  // input in Farenheit
+  if (inputUnit.value === "F") {
+    let outputC = fToC(inputTemp.value);
+    let outputK = fToK(inputTemp.value);
+    displayOne(outputC);
+    displayTwo(outputK);
+  } else 
+  // input in Celsius
+  if (inputUnit.value === "C") {
+    let outputF = cToF(inputTemp.value);
+    let outputK = cToK(inputTemp.value);
+    displayOne(outputF);
+    displayTwo(outputK);
+  } else 
+  // input in Kelvin
+  if (inputUnit.value === "K") {
+    let outputF = kToF(inputTemp.value);
+    let outputC = kToC(inputTemp.value);
+    displayOne(outputF);
+    displayTwo(outputC);
+  }
+
+  // message logic
+  if (inputUnit.value === "K" && inputUnit.value <= 273.15 || inputUnit.value === "F" && inputUnit.value <= 32 || inputUnit.value === "C" && inputUnit.value <= 0) {
```

Comment:
you have pretty long conditions, the next step is to consider refactoring them into a switch case :D